src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -921,17 +921,12 @@ def __init__(self, test_method):
         self.vm_names = ['cli-test-vm2']
         self.vm_image = 'UbuntuLTS'
         self.auth_type = 'ssh'
-        self.pub_ssh_filename = None
 
     def test_vm_create_ubuntu(self):
         self.execute()
 
     def set_up(self):
         super(VMCreateUbuntuScenarioTest, self).set_up()
-        _, pathname = tempfile.mkstemp()
-        with open(pathname, 'w') as key_file:
-            key_file.write(TEST_SSH_KEY_PUB)
-        self.pub_ssh_filename = pathname
```

Comment:
This fixes an existing bug that under playback, the `pub_ssh_filename` is always None, so the `vm create` ends up getting key value of `'None'`. With my change, this test bugs surfaces with an exception